themes/qgis-theme/static/qgis-style.css

```
@@ -867,6 +867,12 @@ section.menu nav.subnav ul li a:hover {
   background-position: 15px 8px;
   background-repeat: no-repeat;
 }
+/* theming of warning boxes class=".admonition .tip" */
```

```suggestion
/* theming of warning boxes class=".admonition .important" */
```
So now it'd look like warning, right?
For my knowledge, do you copy paste the changes from css <-->less files or is there any command you could run to update?

---

> So now it'd look like warning, right?

yes, but the icon can be changed later if we need to.

> For my knowledge, do you copy paste the changes from css <-->less files or is there any command you could run to update?

I did compile the CSS from ".less" sources, I think it's the right thing to do.